wpimath/src/test/java/edu/wpi/first/math/interpolation/InterpolatingMeasuresTreeMapTest.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.interpolation;
+
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.measure.Distance;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class InterpolatingMeasuresTreeMapTest {
```

Comment:
In general we stick to one test per class, even if the class is generic. Since these tests are just testing `InterpolatingTreeMap`, we should move these tests into `InterpolatingTreeMapTest`.
Note that `InterpolatingDoubleTreeMapTest` exists because we have a separate `InterpolatingDoubleTreeMap` class (which only exists because the previous version of `InterpolatingTreeMap` only supported doubles, and we wanted to make the transition a little easier by providing a separate class).